### ISSUE TITLE
Action myoclonus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7528]
**Preferred term label:**
Action myoclonus

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
616230 
PMID: 24782409  4/4
PMID: 30800706  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
